### PR TITLE
article about glassmorphism on android

### PR DESCRIPTION
**Pull request by othimar**
This is an android explaining how to achieve glassmorphism on android using Blurview Library.

### CODE REVIEW

**Comment by tericcabrel** [comment]
Hello @othimar!
Thank you for your contribution!
Please, can you delete the `package-lock.json` file, as it is not needed here?

**Comment by othimar** [comment]
Hello Sir. Sorry, I forgot to remove `package-lock.json`. I'm doing it now.

File: data/blog/introduction-to-glassmorphism-on-android.mdx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+Then add **BlurView** inside the file **build.gradle**, in the **app** folder.
+
+```gradle
+implementation 'com.github.Dimezis:BlurView:version-2.0.3'
+```
+
+Now, we can start coding.
+
+## let's code
```
Comment (tericcabrel) [inline data/blog/introduction-to-glassmorphism-on-android.mdx]: This headline is not necessary. You can directly put the next headline `## Add a background to the layout`

File: data/blog/introduction-to-glassmorphism-on-android.mdx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+authors: ['othimar']
+---
+
+# Overview
+
+<TOCInline toc={props.toc} exclude="Overview" toHeading={3} />
+
+# What is glassmorphism?
+
+Glassmorphism is a design style that uses transparency and blur to create glass-like effect . For transparency, we can use a transparent color. For the blur effect, we will use **[BlurView](https://github.com/Dimezis/BlurView)** library.
```
Comment (tericcabrel) [inline data/blog/introduction-to-glassmorphism-on-android.mdx]: > Glassmorphism is a design style that uses transparency and blur to create glass-like effect .

- There is a whitespace before the full stop that should be deleted.
- `to create a glass-like effect` or  `to create glass-like effects` choose what sounds good for you.

**Comment by tericcabrel** [comment]
Hey @othimar!
Please, rebase your branch to main. I fixed the GitHub actions, so it will work know.

**Comment by tericcabrel** [comment]
@othimar there is an issue with the `yarn.lock` file. Please, can you delete it and run `yarn install` again?

**Comment by othimar** [comment]
I deleted the `yarn.lock` file. But I'm getting errors every time I run `yarn install`. I even cloned the original project but I keep getting those errors. I'm going to try again tomorrow.

**Pull request closed without merging**